### ISSUE TITLE
Updating multiple lines with `name` does not work

### ISSUE THREAD

**Issue by SsnL** [facebookresearch/visdom#596]
**Bug Description**
Updating a `vis.line` with `update='append'` does not work if multiple lines are being updated and `name=` is specified.

This works (multiple lines without `name=`): 
```py
In [28]: vis.line(np.random.randn(1, 10), [0], win='line', update='append')
Out[28]: 'line'

In [29]: vis.line(np.random.randn(1, 10), [1], win='line', update='append')
Out[29]: 'line'
```

![image](https://user-images.githubusercontent.com/5674597/55645260-1b8e2980-57a6-11e9-95fc-319694b1ef6a.png)

This works (one line with `name=`):
```py
In [35]: vis.line(np.random.randn(1), [0], win='line', update='append', name='name')
Out[35]: 'line'

In [36]: vis.line(np.random.randn(1), [1], win='line', update='append', name='name')
Out[36]: 'line'
```

![image](https://user-images.githubusercontent.com/5674597/55645341-52643f80-57a6-11e9-869f-f7dbe1d185d2.png)

This **doesn't** work (multiple lines with `name=`):
```py
In [38]: vis.line(np.random.randn(1, 10), [0], win='line', update='append', name='name')
Out[38]: 'line'

In [39]: vis.line(np.random.randn(1, 10), [1], win='line', update='append', name='name')
Out[39]: ''

In [40]: vis.line(np.random.randn(1, 10), [2], win='line', update='append', name='name')
Out[40]: ''
```
Only the first set of values are displayed:
![image](https://user-images.githubusercontent.com/5674597/55645440-97887180-57a6-11e9-9abd-eec21f12a239.png)

**Comment by JackUrb** [comment]
This is as expected - multiple lines with a single specified name is undefined behavior for visdom, as we don't yet support specifying multiple names at once. What would you expect the behavior to be if you're specifying the name of a single trace but supplying multiple traces worth of data?

**Comment by SsnL** [comment]
@JackUrb In that case I expect an error. :)

**Comment by JackUrb** [comment]
Seems reasonable to me - would you like to add a check to `line` and open a PR?

**Comment by SsnL** [comment]
@JackUrb Sure. Should this check go into `line` or `scatter`?

Btw, is there a way to specify multiple line names with a single call to `line`?

**Comment by JackUrb** [comment]
Both rely on `scatter` so putting it there would take care of both. As of now there isn't a way to specify multiple names though. I imagine the correct implementation would be taking an array of names, ensuring it's the same length as the number of traces added in, and then looping over the `server.py` update that handles named updates for each of the names.

**Comment by SsnL** [comment]
closed via #597
